src/batVect.ml

```diff
@@ -1165,4 +1197,36 @@ struct
   let print ?(first="[|") ?(last="|]") ?(sep="; ") print_a out t =
     BatEnum.print ~first ~last ~sep print_a out (enum t)
 
+  module Labels =
+  struct
+    let init n ~f      = init n f
+    let concat ~v1 ~v2 = concat v1 v2
```

Comment:
I believe that the labels are not useful nor very informative for `concat`; the spatial position of the arguments should tell us everything we need to know.